tutorials/grid_coordinates.py

```diff
@@ -0,0 +1,324 @@
+"""
+.. _grid_coordinates:
+
+Gridding coordinates
+====================
+
+Grid coordinates in Verde are used to create points on a regularly spaced grid
+that are then used in the spline method to interpolate between sample data
+points. The grid can be specified either by the number of points in each
+dimension (the *shape*) or by the grid node spacing.
+
+Creating regular grids in Verde is done using the :func:`verde.grid_coordinates`

```

Comment:
I wouldn't mention the adjusting of region or spacing until next section. So I would simplify this paragraph as follows:
```
Creating regular grids in Verde is done using the :func:`verde.grid_coordinates`
function. It creates a set of regulary spaced points in both the west-east and
south-north directions, i.e. a two-dimensional spatial grid.
```

Reply:
I have simplified this section and moved adjustments to the next sections

Reply:
I agree with @santisoler. We probably don't need the first paragraph. One more distinction, `grid_coordinates` makes the *coordinates* for a grid, not the grid iteself.